### PR TITLE
release(v0.1.0-beta): update changelog

### PR DESCRIPTION
**Pull request by rhamzeh**
## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## v0.1.0-beta
+
+### [0.1.0-beta](https://github.com/openfga/go-sdk/releases/tag/v0.1.0-beta) (2023-07-11)
+
+Initial OpenFGA CLI release
+- Support for [OpenFGA](https://github.com/openfga/openfga) API
+  - CRUD stores
+  - Create, read, list and validate authorization models
+  - Writing, Deleting Reading and Importing Tuples
```
Comment (adriantam) [inline CHANGELOG.md:11]:
```suggestion
  - Writing, deleting, reading and importing tuples
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - Reading Tuple changes
```
Comment (adriantam) [inline CHANGELOG.md:12]:
```suggestion
  - Reading tuple changes
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+## v0.1.0-beta
+
+### [0.1.0-beta](https://github.com/openfga/go-sdk/releases/tag/v0.1.0-beta) (2023-07-11)
+
+Initial OpenFGA CLI release
+- Support for [OpenFGA](https://github.com/openfga/openfga) API
+  - CRUD stores
```
Comment (adriantam) [inline CHANGELOG.md:9]:
```suggestion
  - Create, get, delete and list stores
```
as we don't support update

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  - Create, read, list and validate authorization models
+  - Writing, Deleting Reading and Importing Tuples
+  - Reading Tuple changes
+  - Authorization checks
+  - Listing objects a user has access to
+  - Listing relations a user has on an object
+  - Using Expand to understand why access was granted
```
Comment (adriantam) [inline CHANGELOG.md:16]:
```suggestion
  - Using expand to understand why access was granted
```

**Review by adriantam — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rhamzeh) [reply CHANGELOG.md]: Changed

File: CHANGELOG.md (hunk 2 above)
Comment (rhamzeh) [reply CHANGELOG.md]: Changed

File: CHANGELOG.md (hunk 3 above)
Comment (rhamzeh) [reply CHANGELOG.md]: Changed

File: CHANGELOG.md (hunk 4 above)
Comment (rhamzeh) [reply CHANGELOG.md]: Changed

**Review by adriantam — APPROVED** [review]

**Pull request merged**